tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_BlastWave.cpp

```diff
@@ -106,13 +124,16 @@ void test_variables(const DataType& used_for_size) noexcept {
   const std::array<double, 3> magnetic_field{{0.1, 0.0, 0.0}};
   const double adiabatic_index = 1.3333333333333333333;
 
+  // last argument is 1.0 if Cylindrical and 0.0 if Spherical geometry
   const auto member_variables = std::make_tuple(
       inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
-      outer_pressure, magnetic_field, adiabatic_index);
+      outer_pressure, magnetic_field, adiabatic_index,
+      geometry == grmhd::AnalyticData::BlastWave::Geometry::Cylindrical ? 1.0
+                                                                        : 0.0);
 
   BlastWaveProxy cylindrical_blast_wave(
       inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
-      outer_pressure, magnetic_field, adiabatic_index);
+      outer_pressure, magnetic_field, adiabatic_index, geometry);
```

Comment:
if `geometry` is specifiable, the variable should be renamed `cylindrical_blast_wave` -> `blast_wave`